pkg/applicationserver/config.go

```diff
@@ -65,33 +65,47 @@ type EndDeviceFetcherCacheConfig struct {
 type EndDeviceFetcherCircuitBreakerConfig struct {
 	Enable    bool          `name:"enable" description:"Enable circuit breaker behavior on burst errors"`
 	Timeout   time.Duration `name:"timeout" description:"Timeout after which the circuit breaker closes"`
-	Threshold int           `name:"threshold" description:"Number of failed fetching attempts after which the circuit breaker opens"`
+	Threshold int           `name:"threshold" description:"Number of failed fetching attempts after which the circuit breaker opens"` // nolint:lll
 }
 
 // EndDeviceMetadataStorageConfig represents the configuration of end device metadata operations.
 type EndDeviceMetadataStorageConfig struct {
+	// DEPRECATED: use the metadata.EndDeviceRegistry for location storage.
 	Location EndDeviceLocationStorageConfig `name:"location"`
```

Comment:
Add `DEPRECATED` also to the description. See other deprecated config for reference.

Reply:
This struct is still used, only the Location field is not used anymore. If I do a new struct the name will be similar because it's still an end device metadata registry which seems confusing to have. I chose to just mark that one field as deprecated instead and add the new fields to the same structure.

Reply:
I meant the `description:"..."` tag

Reply:
got it, fixed